rero_ils/modules/acq_order_lines/jsonschemas/acq_order_lines/acq_order_line-v0.0.1.json

```diff
@@ -0,0 +1,160 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "AcqOrderLine",
+  "description": "JSON schema for acq_order_line.",
+  "additionalProperties": false,
+  "propertiesOrder": [
+    "acq_account",
+    "description",
+    "order_line_status",
+    "quantity",
+    "amount",
+    "discount_amount",
+    "rate",
+    "note"
+  ],
+  "required": [
+    "$schema",
+    "pid",
+    "acq_account",
+    "acq_order",
+    "amount",
+    "description"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate acq_order_line records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://ils.rero.ch/schema/acq_order_lines/acq_order_line-v0.0.1.json"
+    },
+    "pid": {
+      "title": "AcqOrderLine ID",
+      "type": "string",
+      "minLength": 1
+    },
+    "description": {
+      "title": "Description",
+      "type": "string",
+      "minLength": 1
+    },
+    "order_line_status": {
+      "title": "Order line status",
+      "type": "string",
+      "enum": [
+        "approved",
+        "canceled",
+        "ordered",
+        "requested",
+        "pending",
+        "received"
+      ],
+      "default": "pending",
+      "form": {
+        "options": [
+          {
+            "value": "approved",
+            "label": "Approved"
+          },
+          {
+            "value": "canceled",
+            "label": "Canceled"
+          },
+          {
+            "value": "ordered",
+            "label": "Ordered"
+          },
+          {
+            "value": "requested",
+            "label": "Requested"
+          },
+          {
+            "value": "pending",
+            "label": "Pending"
+          },
+          {
+            "value": "received",
+            "label": "Received"
+          }
+        ]
+      }
+    },
+    "quantity": {
+      "title": "Quantity",
+      "type": "number",
```

Comment:
of interest to use `integer` instead of `number` ? integer limit to non-floating number.

Reply:
yes, of course!